packages/backend/src/database/migrations/20210722101405_sessions.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable('sessions', (table) => {
+        table.string('sid').notNullable().primary();
+        table.json('sess').notNullable();
```

Comment:
If we ever plan on making this json session indexable we should use `jsonb`. do you think we'll ever store anything else in this session data?